exp-models/addon/services/current-user.js

```diff
@@ -0,0 +1,40 @@
+import Ember from 'ember';
+
+import config from 'ember-get-config';
+
+export default Ember.Service.extend({
+    store: Ember.inject.service(),
+    session: Ember.inject.service(),
+    account: null,
+
+    init() {
+        this._super(...arguments);
+
+        this.get('session').on('invalidationSucceeded', () => {
+            this.set('account', null);
+        });
+    },
+
+    getCurrentUser() {
+        return new Ember.RSVP.Promise((resolve, reject) => {
+            if(!this.get('session.isAuthenticated')) {
+                resolve(null);
+            }
+            else {
+                if (this.get('account')) {
+                    resolve(this.get('account'));
+                }
+
+                var data = this.get('session.data.authenticated');
+                var accountId = null;
+                if(data.provider === 'osf') {
+                    accountId = config.JAMDB.testAccount.id;
```

Comment:
If we're going to embed what is basically a development setting inside production code, I propose one of the following:

- Modify the sample data so that `testAccount` explicitly refers to an administrator, or,
- Create a followup ticket that explicitly prevents administrators from creating new sessions (except perhaps when `--environment` is `development`)